docs/infrastructure/deployment-targets/windows-targets/polling-tentacles-over-port-443.md

```diff
@@ -0,0 +1,74 @@
+
+## Octopus Cloud
+
+The setup of a Polling Tentacle for an [Octopus Cloud](/docs/octopus-cloud/index.md) instance over port 443 is the same as a [Polling Tentacle over port 10943](/docs/infrastructure/deployment-targets/windows-targets/tentacle-communication.md#polling-tentacles), except when registering the Tentacle. Change the `register-with` and `register-worker` commands:
+ - Omit the `--server-comms-port` parameter.
+ - Specify the `--server-comms-address <address>` parameter.
+   - The address to use is your [Octopus Cloud](/docs/octopus-cloud/index.md) instance URL prefixed with `polling.` (e.g. `https://polling.<yoururl>.octopus.app`).
+
+### Registering a new Tentacle
+```powershell
+.\Tentacle register-with --instance MyInstance --server "https://<yoururl>.octopus.app" --server-comms-address "https://polling.<yoururl>.octopus.app" --comms-style TentacleActive --apiKey "API-CS0SW5SQJNLUBQCUBPK8LZY3KYO" --environment "Test" --role "Web"
+```
+
+### Changing an existing Tentacle
+
+```powershell
+.\Tentacle service --instance MyInstance --stop
+.\Tentacle configure --reset-trust
+.\Tentacle register-with --instance MyInstance --server "https://<yoururl>.octopus.app" --server-comms-address "https://polling.<yoururl>.octopus.app" --comms-style TentacleActive --apiKey "API-CS0SW5SQJNLUBQCUBPK8LZY3KYO" --environment "Test" --role "Web"
+.\Tentacle service --instance MyInstance --start
+```
+
+### Registering a new Worker
+```powershell
+.\Tentacle register-worker --instance MyInstance --server "https://<yoururl>.octopus.app" --server-comms-address "https://polling.<yoururl>.octopus.app" --comms-style TentacleActive --apiKey "API-CS0SW5SQJNLUBQCUBPK8LZY3KYO" --workerpool MyWorkerPool
+```
+
+### Changing an existing Worker
+```powershell
+.\Tentacle service --instance MyInstance --stop
+.\Tentacle configure --reset-trust
+.\Tentacle register-worker --instance MyInstance --server "https://<yoururl>.octopus.app" --server-comms-address "https://polling.<yoururl>.octopus.app" --comms-style TentacleActive --apiKey "API-CS0SW5SQJNLUBQCUBPK8LZY3KYO" --workerpool MyWorkerPool
+.\Tentacle service --instance MyInstance --start
+```
+
+## Self-hosted
+
+For self-hosted installations of Octopus Server, you will require additional network configuration and/or services to support the use of Polling Tentacles, Octopus Web Portal and REST API all over port 443. 
+
+A reverse proxy can be set up either on the machine or a machine/appliance that fronts it. The reverse proxy would inspect connections coming in on the same port and decide which backend port to forward them to.
+The proxy could differentiate the connections based on:
+- Hostname (TLS SNI)
+- IP Address
+
+This reverse proxy must pass-through all Tentacle traffic as [SSL offloading is not supported](/docs/infrastructure/deployment-targets/windows-targets/tentacle-communication.md#ssl-offloading-is-not-supported).
```

Comment:
Just a thought, an example might be nice here (say NGINX).

Reply:
👍 We have updated the sentence to include "(e.g. NGINX)" however it was deemed out-of-scope for the project to include specific sample reverse proxy configs.